server/app/services/payment_group_affiliations_creator.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+class PaymentGroupAffiliationsCreator
+  attr_accessor :payment_group, :payment_affiliations
+
+  def initialize(management_group:, payment_group_params:, payment_affiliations_params:)
+    @management_group = management_group
+    @payment_group_params = payment_group_params
+    @payment_affiliations_params = payment_affiliations_params
+  end
+
+  def call
```

Comment:
```suggestion
  def call!
```

の方が例外吐くことわかりやすくて良さそう。他で使ったときに例外吐くこと知らないと困惑するかもだし。